test-data/src/chain_builder.rs

```diff
@@ -49,6 +50,12 @@ impl Into<IndexedTransaction> for TransactionBuilder {
 }
 
 impl TransactionBuilder {
+	pub fn overwintered() -> TransactionBuilder {
```

Comment:
why not setting version as well?

Reply:
These are exposed as two separate fields in the `Transaction` structure. It makes no sense to set `tx.version` when we're setting `tx.overwintered`. It could be useful if we'd expose these fields only through `Transaction::` methods, but that isn't the case

Reply:
nvm, but you always do `overwintered().set_version(2)` in tests why not combining to be aligned more with "convention over configuration"